jsonschema/MsgPoseRelative.json

```diff
@@ -0,0 +1,66 @@
+{
+  "copyright": [
```

Comment:
Is this msg part of this PR as well?

Reply:
I think someone forgot to run all the code gen at some point, and missed some for this message - https://github.com/swift-nav/libsbp/blob/19bd1624d27a47afcaf35cb90fb69f538e88f9d5/spec/yaml/swiftnav/sbp/navigation.yaml#L2559